community/kernel/src/main/java/org/neo4j/kernel/DatabaseAvailability.java

```diff
@@ -63,18 +74,19 @@ public void stop()
     {
         // Database is no longer available for use
         // Deny beginning new transactions
-        availabilityGuard.deny(this);
+        availabilityGuard.deny( this );
 
         // Await transactions stopped
-        awaitNoTransactionsOr( 10_000  /* ms */);
+        awaitNoTransactionsOr();
     }
 
-    private void awaitNoTransactionsOr( int orUntilDeadline )
+    private void awaitNoTransactionsOr()
     {
-        long deadline = SYSTEM_CLOCK.currentTimeMillis() + orUntilDeadline;
-        while ( transactionMonitor.getNumberOfActiveTransactions() > 0 && SYSTEM_CLOCK.currentTimeMillis() < deadline)
+        long deadline = SYSTEM_CLOCK.currentTimeMillis() + awaitActiveTransactionDeadlineMillis;
+        while ( transactionMonitor.getNumberOfActiveTransactions() > 0 && SYSTEM_CLOCK.currentTimeMillis() < deadline )
         {
             Thread.yield();
+            LockSupport.parkNanos( 1_000_1000 );
```

Comment:
Oh I meant to remove the yield call, will do that. Also I thought about that... do you think we should avoid that at all cost? I'm fine with bumping to 10ms though.